blazor/heatmap-chart/selection.md

```diff
@@ -48,4 +56,46 @@ In the [Blazor HeatMap Chart](https://www.syncfusion.com/blazor-components/blazo
 
 ```
 
-![Selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-selection.png)
\ No newline at end of file
+![Selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-selection.png)
+
+The image below illustrates the process of selecting multiple cells in the HeatMap by clicking and dragging the mouse across the specific cells.
```

Comment:
The illustration below shows how to select multiple cells in the HeatMap by clicking and dragging the mouse across the cells.